corehq/apps/reports/standard/tableau.py

```diff
@@ -78,10 +77,11 @@ def tableau_visualization_ajax(request, domain):
     server_name = visualiation_data.pop('server_name')
     validate_hostname = visualiation_data.pop('validate_hostname')
     target_site = visualiation_data.pop('target_site')
-    domain_username = visualiation_data.pop('domain_username')
 
+    # An equivalent Tableau user with the username "HQ/{role name}" and proper permissions must exist.
+    tableau_username = f"HQ/{request.couch_user.get_role(domain).name}"
```

Comment:
It might be worth catching `DomainMembershipError` here and making sure it's displayed in an intelligible way - I think right now it will just 500.